src/Test/PdbUtilities/Shared/SymUnmanagedSequencePoint.cs

```diff
@@ -2,48 +2,10 @@
 
 using System;
 using System.Collections.Immutable;
-using System.Diagnostics;
-using System.Runtime.InteropServices;
 using Microsoft.CodeAnalysis;
 
-namespace Microsoft.VisualStudio.SymReaderInterop
+namespace Microsoft.DiaSymReader
```

Comment:
While I agree that it's strange to use a namespace from another assembly, in this particular case, essentially every consumer of the namespace will also want the extension methods.